src/components/NavBar/NavBar.module.css

```diff
@@ -1,40 +1,62 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap');
 .navBar {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
-  background-color: #262e4b;
+  background-color: #232627;
   width: 100%;
   z-index: 1000;
 }
 
 .navBarContainer {
   display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.8rem;
+  height: 4rem;
 }
 
 .navContainer {
   display: flex;
 }
 
 .navLink {
-  margin: auto 10px;
+  font-family: 'Bai Jamjuree', sans-serif;
```

Comment:
`Bai Jamjuree` should be imported from google fonts. It is not available by defauly